core/src/main/resources/error/error-classes.json

```
@@ -592,6 +592,29 @@
       "Detected an incompatible DataSourceRegister. Please remove the incompatible library from classpath or upgrade it. Error: <message>"
     ]
   },
+  "INCOMPLETE_TYPE_DEFINITION" : {
+    "message" : [
+      "Incomplete complex type:"
+    ],
+    "subClass" : {
+      "ARRAY" : {
+        "message" : [
+          "The definition of `ARRAY` type is incomplete. You must provide an element type. For example: `ARRAY<elementType>`."
```

Please, quote types in the same way as `toSQLType()`.
```suggestion
          "The definition of \"ARRAY\" type is incomplete. You must provide an element type. For example: \"ARRAY<elementType>\"."
```

---

Updated for all three error msg